.gitignore

```diff
@@ -1,6 +1,8 @@
 # build output
 dist
 
+/backend/config/secrets.js
```

Comment:
[nit] Should be labeled with a comment (to match style of other lines)

Reply:
Fixed 👍